### DISCUSSION TITLE
v0.9.1 Generated items and more

### DISCUSSION THREAD

**Discussion by neph1** [neph1/LlamaTale#16] [Announcements]
## What's Changed
* fix char builder by @neph1 in https://github.com/neph1/LlamaTale/pull/14
* parsing generated items by @neph1 in https://github.com/neph1/LlamaTale/pull/15

- Generated items are now parsed and inserted correctly.
- Added a new concept: zone descriptions:
Each zone (area consisting of several locations) can have a description, a list of races commonly populating it, and a list of items which can be found. These are added to the prompt the LLM receives to give it a hint of what it can use. This will be expanded upon later.
- I managed to get the LLM to more consistently generate locations, it's also quite generous with items. Maybe need to pass some probability from the MUD whether it should generate items and mobs.
- I added a flag so that locations created by the LLM  don't receive a generated description when entering them. This to avoid a double inference load. I also figure since they're already procedurally generated, they don't need it as much as the 'static' locations. Do you agree? If not, let's discuss it! I may make it configurable.

If you get errors when generating content, that is not JSON related (those are generally 'Unterminated string' or 'Expecting delimiter'), I'm interested in receiving error reports. Copying the generated content from the LLM window would be helpful. I've tried to compensate for weird stuff the LLM generates, but there are probably many more cases.

**Full Changelog**: https://github.com/neph1/LlamaTale/compare/v0.9.0.1...v0.9.1

<hr /><em>This discussion was created from the release <a href='https://github.com/neph1/LlamaTale/releases/tag/v0.9.1'>v0.9.1 Generated items and more</a>.</em>

**Comment by kroryan** [comment]
Hii!! I think u should include packaging on requirements.txt it was asking for it this New version
